src/shared/logger/universalFileTransport.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import winston from 'winston'
+import TransportStream from 'winston-transport'
+import { FileSystemCommon } from '../../srcShared/fs'
+import * as vscode from 'vscode'
+
+/**
+ * Inspired from: https://github.com/MarcoMedrano/winston-transport-browserconsole/blob/master/src/lib/BrowserConsole.ts
+ */
+export class UniversalFileTransport extends TransportStream {
```

Comment:
Let's stick to "shared" where we mean "universal", to avoid multiple synonyms. But also generally, anything in /shared/ is intended to be universal so we don't need to mention it explicitly in the class name.
```suggestion
export class FileTransport extends TransportStream {
```